requirements/base.txt

```diff
@@ -1,23 +1,53 @@
-# edX Internal Requirements
-edx-i18n-tools
-edx-submissions>=2.0.12
-git+https://github.com/edx/django-rest-framework.git@1ceda7c086fddffd1c440cc86856441bbf0bd9cb#egg=djangorestframework==3.6.3
-git+https://github.com/edx/XBlock.git@xblock-1.0.1#egg=XBlock==1.0.1
-git+https://github.com/edx/xblock-sdk.git@331e50bed2a3c9014d998cc4428e92bf658baf8d#egg=xblock-sdk==0.1.6
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    make upgrade
+#
+--index-url http://edx.devstack.devpi:3141/root/pypi/+simple/
```

Comment:
I ran into issues with edx-proctoring when these flags got included in the *.txt files.  If this build fails, that might be the reason.  We should look into what edx-platform does about this index/host stuff.